core-java-modules/core-java-os/pom.xml

```diff
@@ -67,6 +67,7 @@
         <collections-generic.version>4.01</collections-generic.version>
         <unix4j.version>0.4</unix4j.version>
         <grep4j.version>1.8.7</grep4j.version>
+        <mockito.version>4.4.0</mockito.version>
```

Comment:
Why do we need to specify the version here?

Reply:
Yes, this is a quick fix to run the build. Once the build passes we can investigate the code changes required with the new version.

Reply:
OK, I will create separate task for this module.

Reply:
https://team.baeldung.com/browse/JAVA-33816